.github/workflows/integration-test.yml

```diff
@@ -33,14 +33,14 @@ jobs:
 
       - name: Upload debug transaction logs
         if: always()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         with:
           name: test-transacion-logs
           path: tmp
 
       - name: Collect docker logs on failure
         if: failure()
-        uses: jwalton/gh-docker-logs@v1
+        uses: jwalton/gh-docker-logs@v2
```

Comment:
I would recommend pinning actions to a commit sha instead of tags, as per the recommendations in https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Tracking this on https://github.com/HathorNetwork/hathor-wallet-lib/issues/493